_e2e/raw2science.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+export FINKCONFIG="$DIR"
+
+mkdir -p $HOME/.kube
+cat >> $HOME/.kube/config << EOF
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMvakNDQWVhZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJME1ERXhPVEUxTlRNeE1sb1hEVE0wTURFeE5qRTFOVE14TWxvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTW5YCms4V0oraFZOZFB5NDhaWFp4UHpua2gvbGVWTFJONi9xL1gxM3JOWmRpaVY3OUJYeFNTc1dSem1CUGk0TGhEU0EKMnlSVjF2aW5xVGVzampZZWJYTldFdTV0M24vcEQ5Vk0ySmVPSkg5T0tKOWd2TU80OUVkamFLWldsRDk3eG4rTApkYU9ZQStSeENjcExkN0Rwang2NmlScm81S2Q0czh2MEthaUVPTktESFNSQ1V1Z1JYZ0J3UldLcDBNZ0tsK3R4CkdBVEY0Sk83WXoxUzVKR0pVRmozR1ViSmZrWHlIVDUxTWN2YmFnamIzTkRiRWpZNnJPVExKanE5KzY5T1VBQ0UKTUdzdFo5b0puOXJTdVNBeFN1V3BCZ3BYVkkyLzd5WVZGZW9WUkZWenAyenhGZVc0V3l6YnRGT09Ea09kMk5ENAp4c21tSGVBN0FkUHo2cStiT3pjQ0F3RUFBYU5aTUZjd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0hRWURWUjBPQkJZRUZEdmdVdU1oKzhoYkNyVkI4eDJkOFdvUjJ1cm1NQlVHQTFVZEVRUU8KTUF5Q0NtdDFZbVZ5Ym1WMFpYTXdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUJBRDlGSXZkLzlZQytVaEltS0VJagpqK2RVY3RYYlorZjU5clRBU1l1WDFPUCtwbmVhNVNTcnR5Y0RLTUxPZ2doL3BHY0xpQW5KMnJYWmlTTG40eEVkCmtWL2xKaE92T0JvK0VqbmhzUkZLRkI5MW5La0toVWt0ejFUL0FJeFBCSm5RT3NEemF1UTN1NTk2bW94eHh5OUkKeTRWT0FpbVltbHRRclVtR3Y5bVlEaEJaMUlPRkZuS3gxK0daNW9YRllqQVo3aTRZK3hNZkRYT21td0pjTEVBTAo3Z0duUjBkOFNGVUFaSitqU3NzMVoyMFdFbXNyVC9ndFJ1UDFWQXpDajlXMEtJZ1BBZnUzQnJTdHhUY2FxcTlyClhpdE5mY3I4UXordDMzbjMrenBObUV6NUNPWDBMVmVlSE0yNkpDTjlZOHh2eElLVThBL2VsTS9waUlra0V4TysKNkNJPQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
Just to understand, where this certificate comes from and how it has been generated?

Reply:
Good point, this certificate is useless for e2e test. I have removed it.

Reply:
thanks!